benchmark/README.md

```diff
@@ -52,7 +52,18 @@ Run a specific benchmark in `Prelude.Serial` suite:
 $ ./bench.sh --benchmarks Prelude.Serial -- Prelude.Serial/o-1-space/generation/unfoldr
 ```
 
-## Comparing results for regression
+Run a benchmark directly instead of running it through `bench.sh`:
+
+```
+$ cabal run bench:Prelude.Serial -- --quick Prelude.Serial/o-1-space/generation/unfoldr
+```
+
+The options after `--` are the `gauge` options.
```

Comment:
`--stream-size` is not a gauge option. We may add multiple options specific to streamly in the future. Maybe phrase this differently?